llvm/test/CodeGen/AArch64/avoid-pre-trunc.ll

```diff
@@ -0,0 +1,89 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc -mtriple=aarch64 < %s | FileCheck %s
+
+define i32 @lower_lshr(<4 x i32> %a, <4 x i32> %b, <4 x i32> %c, <4 x i32> %d, <4 x i32> %e, <4 x i32> %f, <4 x i32> %g, <4 x i32> %h) local_unnamed_addr #0 {
```

Comment:
Don't need local_unnamed_addr and the attributes

Reply:
My bad for not spotting it, I've removed it now.